app/components/donor-conditions.js

```diff
@@ -1,14 +1,15 @@
-import Ember from 'ember';
+import Ember from "ember";
 
 export default Ember.Component.extend({
   store: Ember.inject.service(),
 
-  donorConditions: Ember.computed(function(){
-    return this.get('store').peekAll('donor_condition');
+  donorConditions: Ember.computed(function() {
```

Comment:
Everything in this repo is shared between admin and donor (not in a good way).

Are we sure that component is only used on the Donor app ?

Reply:
It is used in both Donor and Admin app.